### ISSUE TITLE
Wish:  Player active indicator

### ISSUE THREAD

**Issue by dlister70** [mainIine/foe-helfer-extension#1033]
When looking for sniping opportunities in a neighborhood, it is helpful to know if that player is dormant or has been active recently.  Will your points just sit on a dormant building because the player is no longer playing the game, or is this an active player who is still possibly leveling their buildings?  

Possible indicators of an active player could be:

1.  If they have aided or attacked in your event history.
2.  If they have donated to your great buildings in the past week.  
3.  If you've visited the player and their collections have changed since you last visited (example:  The last time you were in their city, there were goods to plunder but now the goods are picked up, or in a different amount).  
4.  If you've looked at their great buildings and their forge point contributions have changed since the last time that you looked.  

There could perhaps be a "Player last active 3 days ago." indicator within the sniping calculator.

**Comment by scibuff** [comment]
TL;DR the GB Cost Calculator already does it (in a way)

4) is already done - the "potential investment/snipe" window from the GB cost calculator lists only buildings that have changed since your last "visit". So if you check your friend list every day (or every other day) it will show only the "delta" since the last time you looked

1) I believe you'd have to go through all pages of your event history to get the data
2) I believe you'd have to visit every one of your GBs to get the data
3) I believe you'd have to the player's city

**Comment by andrgin** [comment]
1.) Most of myneighbors do not attack or help me because it is not worth the effort. To be honest the only reason I help my neighbors is because I am going through the list for sniping anyway and this is more a bookmark whom I have already checked for sniping today.

2.) It is very rare that a neighbor invests in my great buildings because I do leave places below 1.9 open. So this does not tell me anything.

3.) What if you do not visit your neighbors for attacking. Data would be awfully incomplete in most cases and would require lots of space for that.

4.) If a line in the overview is green this means there was no investment since last opening the GB. But in reality this does not help very much because good snipes dont live longer than 1-2 days anyway. There no 200+ profit snipes sitting there for months.

The more important thing is to not invest in great buildings that will not be leveled within a few days. A few examples:
.) Arcs above lvl80 are dangerour, below lvl80 it is a very safe investment
.) All other GBs are dangerous at lvl60+. See that sweet snipe on a CdM lvl70? Dont do it as the owner might stop leveling this one.
.) P2+ are always safe if there are already friendly P1/2 investors.
.) Avoid GBs that do not have an investment of the owner yet or only small amounts like 20FP if you dont know if he is currently leveling it.
.) Avoid players doing many GBs at once (for example 10 orange lines on a neighbor you checked yesterday) if rate is not that good. I am still doing a 160% on them, but not a 180+
.) If you have a player where you have to wait 1 month+ for your reward remember their name and avoid them in the future.

If in doubt you can also check guild name. If a player has no guild chances are high he is inactive.

**Comment by dlister70** [comment]
These are all good tips, thank you!

I am currently working on an Iron Age city.  There are a lot of Oracle of Delphi in the neighborhood since that is an early quest that most players complete.  However, since it's such an early quest, they may quit before I get a chance to get my points back.  That's where the idea came from.

I didn't realize that the window was colored differently depending on if there has been an investment since the last time that you looked.  If I'm understanding you correctly, in this image the Oracle of Delphi is listed as green because the owner of the building has not put any points in since the last time that I looked?  What color would it be if they had put points in since last time?

Example picture:  https://i.imgur.com/mEFx9th.png

Thank you for taking the time to respond!

**Comment by andrgin** [comment]
If he puts FP into the GB the next time you open the overview it will show an orange line and profit/rate will be ???, because you need to open the GB again so profit can be calculated.
